src/main/resources/db/migration/V1__Initial_Setup.sql

```diff
@@ -66,3 +66,6 @@ VALUES ('default', "123456543234abdkdkdkd", "TEST TENANT") ;
 
 INSERT INTO `m_sms_bridge` (`tenant_id`, `tenant_phone_no`, `provider_key`, `country_code`, `provider_name`, `description`)
 VALUES (1, '+1234567890', 'Dummy', '+91', 'Dummy SMS Provider - Testing', 'Dummy, just for testing');
+
+INSERT INTO `m_sms_bridge` (`tenant_id`, `tenant_phone_no`, `provider_key`, `country_code`, `provider_name`, `description`)
+VALUES (3, '555-1212', 'telerivet', '+91', 'Telerivet', 'Telerivet Messenger);
```

Comment:
@ankita10r : You should never change existing migration scripts. Write a new migration script to introduce this part.